### PR TITLE
Created the GMN launch event page

### PR DESCRIPTION
**Pull request by rahmot**
Hi! I created the GMN launch event page. Thanks!

### CODE REVIEW

File: content/events/gmn-launch/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+This project goes to show how people-centred the Galaxy community is. Every member will soon have the opportunity to be mentored by topic-based experts and mentors. Professors and students carrying out their projects can work on their projects under the guidance of mentors. This project is a win-win for all participants: Mentors grow in experience, gain new leadership skills and sharpen their expertise, while members can learn about open source, how to create, materials, or even carry out their projects with expert supervision and guidance.
+
+Join us for our exciting online launch event where we will be telling you everything you need to know about the **Galaxy Mentor Network** program and answer any questions you may have as regards the program.
+
+## Agenda:
+
+- Initial Remarks by the Outreachy Mentors (Berenice, Assunta, Beatriz, or Anne) about the Outreachy interns.
+- Introduction to the Galaxy Mentor Network by Outreachy interns (Chinonye and Rahmot)
```
Comment (beatrizserrano) [inline content/events/gmn-launch/index.md:26]:
```suggestion
- Introduction to the Galaxy Mentor Network by Outreachy interns (Chinonye and Rahmot).
```

File: content/events/gmn-launch/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- GMN Website Demo
+- Q & A with the audience
```
Comment (beatrizserrano) [inline content/events/gmn-launch/index.md:28]:
```suggestion
- Q & A with the audience.
```

**Comment by rahmot** [comment]
Thanks, @beatrizserrano! I have applied the sugegstions.

**Review by beatrizserrano — APPROVED** [review]
thanks a lot @rahmot!

**Pull request merged**